lib/gui/app/pages/main/templates/main.tpl.html

```diff
@@ -7,19 +7,28 @@
       </div>
 
       <div class="space-vertical-large">
-        <div ng-if="!main.selection.hasImage()">
-          <button
+          <!-- <button
             class="button button-primary button-brick"
             tabindex="{{ main.selection.hasImage() ? -1 : 1 }}"
             ng-click="image.openImageSelector()">Select image</button>
 
-          <p class="step-footer">
-            {{ ::image.mainSupportedExtensions.join(', ') }}, and
-            <span class="step-footer-underline"
-              uib-tooltip="{{ image.extraSupportedExtensions.join(', ') }}">others</span>
-          </p>
-        </div>
-        <div ng-if="main.selection.hasImage()">
+                      open-image-selector="image.openImageSelector()"-->
+          <image-selector
+            image="main.selection.hasImage()"
+            open-image-selector="image.openImageSelector"
+            main-supported-extensions="image.mainSupportedExtensions"
+            extra-supported-extensions="image.extraSupportedExtensions"
+            show-selected-image-details="main.showSelectedImageDetails"
+            get-image-name="main.selection.getImageName()"
+            get-image-basename="image.getImageBasename()"
+            reselect-image="image.reselectImage"
+            flashing="main.state.isFlashing()"
+            get-image-size="main.selection.getImageSize()"
+
+          >
+          </image-selector>
+
+        <!-- <div ng-if="main.selection.hasImage()">
```

Comment:
Can this just be deleted rather than commented out?

Reply:
I believe it can :)